docs/erp_abrufen_egk.adoc

```diff
@@ -0,0 +1,470 @@
+= E-Rezept API-Dokumentation für Apotheken (Einlösung mit Gesundheitskarte) image:gematik_logo.png[width=150, float="right"]
+// asciidoc settings for DE (German)
+// ==================================
+:imagesdir: ../images
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+:toc: macro
+:toclevels: 3
+:toc-title: Inhaltsverzeichnis
+
+Hier dokumentiert die gematik den Anwendungsfall der Einlösung eines E-Rezepts mittels Stecken der Gesundheitskarte
+
+toc::[]
+
+==  Anwendungsfall Liste einlösbarer E-Rezepte abrufen mit Stecken der Gesundheitskarte
+Mit diesem Anwendungsfall wird die Apotheke durch Übergabe und Stecken der eGK berechtigt, die Liste einlösbarer E-Rezepte vom E-Rezept-Fachdienst abzurufen. Zunächst muss ein VSD-Abruf über den Konnektor erfolgen, der mittels VSDM-Fachmodul eine Online-Gültigkeitsprüfung der eGK durchführt. In der Response gibt der Konnektor die von der eGK gelesenen Versichertenstammdaten VSD zurück.
```

Comment:
bitte um PNW ergänzen:
In der Response gibt der Konnektor die von der eGK gelesenen Versichertenstammdaten VSD und den Prüfungsnachweis zurück.

Reply:
done